### ISSUE TITLE
feature: allow `keeper mysql` to connect with some options from record

### ISSUE THREAD

**Issue by flybyray** [Keeper-Security/Commander#716]
There are multiple use cases where databaseCredentials and serverCredentials  records are not enough to establish a connection.

* connection requires client-ssl certificates and maybe even ca certificate for server certificate validation.
    * extract those from attachments and specify them as args
    ```shell
    --ssl-ca=./securetemp/ca.pem
    --ssl-cert=./securetemp/client-cert.pem
    --ssl-key=./securetemp/client-key.pem
    ```
* connection needs to be established to one of many hosts
    * since 8.0.22 a new option allows to connect to DNS SRV https://dev.mysql.com/doc/refman/8.0/en/mysql-command-options.html#option_mysql_dns-srv-name allow this instead of just host. imagine this is a clustered percona xtradb cluster or something similar.
    
to make that work and easier managable maybe require the user to prepare a `.my.cnf` file ( localy or even this as attachement ).
then make use of `my_print_defaults` 

```shell
my_print_defaults --defaults-file=./securetemp/.my.cnf client mysql
```
this will output each option into one line
```txt
--user=dummy
--password=*****
--dns-srv-name=_mysql._tcp.example.com
--ssl-ca=./securetemp/ca.pem
--ssl-cert=./securetemp/client-cert.pem
--ssl-key=./securetemp/client-key.pem
```
this would allow to configure databaseCredentials and serverCredentials  records for a lot of use cases.

**Comment by sk-keeper** [comment]
Thank you for the feedback. Yes, `mysql` command needs to support additional parameters.
The Commander team is going to address this issue by:
1. `mysql` Commander's command accepts all parameters and passes unsupported parameters to `mysql` program. Providing that this change is simple and safe.
3. `mysql` Commander's command will be reading the Commander configuration file for additional options.
`config.json`:
```
...
   "mysql": "--dns-srv-name=_mysql._tcp.example.com --ssl-ca=./securetemp/ca.pem"
...
```
4. Keeper record may have `mysql:parameters` custom field that contains additional parameters. In this case, custom field value can define placeholders:
${file:attachment} record file attachment is passed by name
${custom_field_name} custom field value

Currently, `mysql` command passes `-u`, `-h`, `-P` parameters
The password is set to `MYSQL_PWD` environment variable. 

I don't know if `mysql-client` app works with `ssh-agent`. If it does then `mysql` Commander's command may store ssh keys to `ssh-agent`.

**Comment by sk-keeper** [comment]
Released in 16.6.0
